pepper-apis/dss-core/src/test/java/org/broadinstitute/ddp/route/IntegrationTestSuite.java

```diff
@@ -273,66 +272,32 @@ public static void tearDown() {
     }
 
     public static void tearDownSuiteServer() {
-        // todo(yufeng) figure out how to shutdown server running in separate process
         DataDonationPlatform.shutdown();
         TransactionWrapper.reset();
     }
 
     private static void bootAppServer(boolean isCacheDisabled) {
         Config cfg = RouteTestUtil.getConfig();
         int port = cfg.getInt(ConfigFile.PORT);
-        boolean spawnProcess = cfg.getBoolean(ConfigFile.BOOT_TEST_APP_IN_SEPARATE_PROCESS);
-
         Map<String, String> serverArgs = new HashMap<>();
         serverArgs.put("config.file", RouteTestUtil.getConfigFilePath());
         serverArgs.put("cachingDisabled", isCacheDisabled + "");
 
-        int bootTimeoutSeconds = 30;
         log.info("Starting app on port {}", port);
 
-        if (spawnProcess) {
```

Comment:
Seems like nobody is using the separate process stuff so I'm removing it to simplify the code.